snowpack/src/types/snowpack.ts

```diff
@@ -47,7 +48,9 @@ export interface PluginRunOptions {
 }
 
 /** map of extensions -> code (e.g. { ".js": "[code]", ".css": "[code]" }) */
-export type PluginLoadResult = string | SnowpackBuildMap;
+export type PluginLoadResult = SnowpackBuildMap;
+
+export type PluginTransformResult = {contents: string, map: string | RawSourceMap};
```

Comment:
I added the correct type to the `map` -- it didn't break any tests, and I don't think it will break any clients since this is a brand new return type that just augments what was previously allowed.  Or did you have some other issue in mind?

Reply:
Awesome! Yeah this is much better than `any`; I just wasn’t sure what that type should be off the top of my head 🙂